tests/python_package_test/test_sklearn.py

```diff
@@ -79,6 +79,14 @@ def multi_error(y_true, y_pred):
 def multi_logloss(y_true, y_pred):
     return np.mean([-math.log(y_pred[i][y]) for i, y in enumerate(y_true)])
 
+def test_binary_fairgbm():
```

Comment:
can we add a short test to assert that calling fairgbm.fit without the constraint_group fails? (should be a TypeError)